set.mm

```diff
@@ -318339,47 +318348,56 @@ any of the balls (i.e. it is in the intersection of the closures).
 
   ${
     $d f x D $.  $d f x J $.  $d f x X $.  $d f x Y $.
-    cmetss.2 $e |- J = ( MetOpen ` D ) $.
+    metsscmetcld.j $e |- J = ( MetOpen ` D ) $.
+    $( A complete subspace of a metric space is closed in the parent space.
+       Formerly part of proof for ~ cmetss .  (Contributed by NM, 28-Jan-2008.)
+       (Revised by Mario Carneiro, 15-Oct-2015.)  (Revised by AV,
+       9-Oct-2022.) $)
+    metsscmetcld $p |- ( ( D e. ( Met ` X )
+                           /\ ( D |` ( Y X. Y ) ) e. ( CMet ` Y ) )
+                         -> Y e. ( Clsd ` J ) ) $=
```

Comment:
Is it useful to have this implication in a separate theorem, when we have the biconditional in the next theorem ?

Reply:
The biconditional requires ``D e. ( CMet ` X )``, whereas for this implication ``D e. ( Met ` X )`` is sufficient.

Reply:
Of course !!! Sorry about that !